src/Take.Blip.Builder/Variables/InputVariableProvider.cs

```diff
@@ -57,7 +57,12 @@ public async Task<string> GetVariableAsync(string name, IContext context, Cancel
 
                 return await GetEntityVariableAsync(input, entityNameAndProperty[1], entityNameAndProperty[2]);
             }
-            
+
+            if (nameToLower.StartsWith("assistant."))
```

Comment:
The `assistant` prefix can be confused with the Google Assistant channel.

Reply:
@nicolasamaduro Maybe you should name the variable `contentAssistant` or something more clear

Reply:
The user will see this prefix on the builder variables library. There is anything related to channels there, so I don't think that such confusion will happen.

Reply:
@liviaalmeida we already use `input.content`, so `input.contentAssistant` can be more confusing.

Reply:
The name "assistant" is not used in any part of the content provider, so it could not be more unrelated.

And there is the collision with the channel name, which will generate confusion. For instance, image if in the future we choose to provide gateway specific variables, this will create a pain for us. Please choose another name.